### DISCUSSION TITLE
Change Subdomain

### DISCUSSION THREAD

**Discussion by dimasandriano** [is-a-dev/register#5335] [Discussion] [answered]
can i change my subdomain? i want to change my subdomain from dimas to dimasandriano

**Comment by wdhdev** [comment, marked as answer]
You can simply rename the file and open a new PR to change your subdomains name.

(Please mark this as the answer if it helped you.)

**Comment by ghost** [comment]
But the old subdomain name will it be removed? 
How to can i remove it?

**Reply by devashish2024** [reply]
If your old username on github was `old` and you changed to `new` and the website was old.is-a.dev, then rename the old 's file to new and make PR, it will rename to new.
